src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/Program.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Sample
+{
+    public class Test
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine ("Hello, World!");
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static int TestMeaning()
+        {
+            int result = ((string)AppContext.GetData("test_runtimeconfig_json")).Equals("25") ? 42 : 1;
```

Comment:
It would be useful to print what `AppContext.GetData` returns here, for debugging.

Reply:
Changed.